srcpkgs/tigervnc/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'tigervnc'
 pkgname=tigervnc
-version=1.10.1
-revision=3
+version=1.12.0
+revision=1
 _xorg_version=1.20.0
 build_style=cmake
-hostmakedepends="automake gettext xorg-util-macros font-util pkg-config libtool xtrans"
-makedepends="fltk-devel zlib-devel libXtst-devel libjpeg-turbo-devel pixman-devel
- xorgproto libxkbfile-devel libXrandr-devel libXfont2-devel gnutls-devel libdrm-devel"
-depends="perl xauth xkeyboard-config"
+hostmakedepends="automake gettext xorg-util-macros font-util pkg-config libtool xtrans pam"
+makedepends="fltk-devel zlib-devel libXtst-devel libjpeg-turbo-devel pixman-devel pam-devel
+ xorgproto libxkbfile-devel libXrandr-devel libXfont2-devel gnutls-devel libdrm-devel
+ libxshmfence-devel"
+depends="perl xauth xkeyboard-config pam"
```

Comment:
Drop `pam` from `depends`; just let `xbps-src` automatically resolve the `pam-libs` dependency.